client/src/components/modal-forms/PhaseDialog.js

```diff
@@ -17,26 +17,32 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-export const NewPhaseDialog = ({ onSubmit, onClose, open }) => {
+export const PhaseDialog = ({ onSubmit, onClose, open, content, isNew }) => {
   const { openToast } = useToast();
   const classes = useStyles();
-  const initialValues = {
-    phaseName: '',
-    startDate: '',
-    endDate: '',
-  };
+  const initialValues = content
+    ? {
+        phaseName: content.name ?? '',
+        startDate: content.start_date ?? '',
+        endDate: content.end_date ?? '',
+      }
+    : {
+        phaseName: '',
+        startDate: '',
+        endDate: '',
+      };
 
-  const handleCreatePhase = async (phase) => {
+  const handleEditPhase = async (phase) => {
```

Comment:
Can this be something more akin to `handleSubmit`?

Reply:
Changed in latest commit.